blsSignatures/blsSignatures.go

```diff
@@ -33,8 +35,8 @@ func GeneratePrivKeyString() (string, error) {
 }
 
 func GenerateKeys() (PublicKey, PrivateKey, error) {
-	fr := bls12381.NewFr()
-	privateKey, err := fr.Rand(cryptorand.Reader)
+	fr := new(fr.Element)
+	privateKey, err := fr.SetRandom()
```

Comment:
This can be simplified to just `new(fr.Element).SetRandom()` to skip creating a separate fr variable

Reply:
Done.